modin/engines/base/frame/data.py

```diff
@@ -244,6 +249,27 @@ def axes(self):
         """The index, columns that can be accessed with an `axis` integer."""
         return [self.index, self.columns]
 
+    def _compute_axis_labels(self, axis: int, partitions=None):
+        """
+        Computes labels for specific `axis`
+
+        Parameters
+        ----------
+        axis : int, axis to compute labels along
```

Comment:
Can axis of string type be passed?

Reply:
no, axis value must be converted to int at the API level